tests/onnx/data/reference_graphs/original_nncf_graph/vgg16.dot

```diff
@@ -0,0 +1,81 @@
+strict digraph  {
+"0 Conv_0" [id=0, type=Conv];
+"1 Relu_1" [id=1, type=Relu];
+"2 Conv_2" [id=2, type=Conv];
+"3 Relu_3" [id=3, type=Relu];
+"4 MaxPool_4" [id=4, type=MaxPool];
+"5 Conv_5" [id=5, type=Conv];
+"6 Relu_6" [id=6, type=Relu];
+"7 Conv_7" [id=7, type=Conv];
+"8 Relu_8" [id=8, type=Relu];
+"9 MaxPool_9" [id=9, type=MaxPool];
+"10 Conv_10" [id=10, type=Conv];
+"11 Relu_11" [id=11, type=Relu];
+"12 Conv_12" [id=12, type=Conv];
+"13 Relu_13" [id=13, type=Relu];
+"14 Conv_14" [id=14, type=Conv];
+"15 Relu_15" [id=15, type=Relu];
+"16 MaxPool_16" [id=16, type=MaxPool];
+"17 Conv_17" [id=17, type=Conv];
+"18 Relu_18" [id=18, type=Relu];
+"19 Conv_19" [id=19, type=Conv];
+"20 Relu_20" [id=20, type=Relu];
+"21 Conv_21" [id=21, type=Conv];
+"22 Relu_22" [id=22, type=Relu];
+"23 MaxPool_23" [id=23, type=MaxPool];
+"24 Conv_24" [id=24, type=Conv];
+"25 Relu_25" [id=25, type=Relu];
+"26 Conv_26" [id=26, type=Conv];
+"27 Relu_27" [id=27, type=Relu];
+"28 Conv_28" [id=28, type=Conv];
+"29 Relu_29" [id=29, type=Relu];
+"30 MaxPool_30" [id=30, type=MaxPool];
+"31 AveragePool_31" [id=31, type=AveragePool];
+"32 Flatten_32" [id=32, type=Flatten];
+"33 Gemm_33" [id=33, type=Gemm];
+"34 Relu_34" [id=34, type=Relu];
+"35 Gemm_35" [id=35, type=Gemm];
+"36 Relu_36" [id=36, type=Relu];
+"37 Gemm_37" [id=37, type=Gemm];
+"38 nncf_model_input_0" [id=38, type=nncf_model_input];
+"39 nncf_model_output_0" [id=39, type=nncf_model_output];
+"0 Conv_0" -> "1 Relu_1"  [label="output_port_id = 0 \n shape = [1, 64, 224, 224] \n input_port_id = 0 \n ", style=solid];
+"1 Relu_1" -> "2 Conv_2"  [label="output_port_id = 0 \n shape = [1, 64, 224, 224] \n input_port_id = 0 \n ", style=solid];
+"2 Conv_2" -> "3 Relu_3"  [label="output_port_id = 0 \n shape = [1, 64, 224, 224] \n input_port_id = 0 \n ", style=solid];
```

Comment:
You should have noticed that something is wrong in your new code that you had to regenerate the entire set of NNCF reference graphs for this change. Hint: it's because you modified the function that generates graphs for the structure analysis, which you shouldn't have done.

Reply:
Of course, I know it. But I did it intentionally. I suppose that such graphs more useful in both ways:
1) For debugging purpose
2) For more accurate graph comparing
What do you think?

Reply:
Both are addressed by a different type of graph, which is the graph received for *visualization*. This type of graph is used for basic structure comparison only and by automatic tests. The extension of the reference file format should be performed in a separate PR, if necessary, so that it is not mistakenly thought of as something that is required for PTQ.